book/02-using-atom/sections/02-text.asc

```diff
@@ -338,7 +338,7 @@ image::images/autocomplete.png[autocomplete]
 
 By default, the autocompleter will look through the current open file for strings that match what you're starting to type.
 
-If you want more options, in the Settings panel for the Autocomplete package you can toggle a setting to make the autocompleter look for strings in all your open buffers rather than just the current file.
+If you want more options, in the Settings panel for the Autocomplete-plus package you can toggle a setting to make the autocompleter look for strings in all your open buffers rather than just the current file.
```

Comment:
We should either refer to the package as Autocomplete Plus or autocomplete-plus, since those are the two naming conventions used in Atom (including the settings-view package).